src/driver/marker_roctx.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef MIGRAPHX_GUARD_RTGLIB_MARKER_ROCTX_HPP
+#define MIGRAPHX_GUARD_RTGLIB_MARKER_ROCTX_HPP
+
+#include <migraphx/marker.hpp>
+#include <migraphx/dynamic_loader.hpp>
+
+namespace migraphx {
+namespace driver {
+inline namespace MIGRAPHX_INLINE_NS {
+
+class marker_roctx
+{
+    private:
```

Comment:
No need for `private:` as the class already defaults to `private`.

Reply:
Done